.github/PULL_REQUEST_TEMPLATE

```diff
@@ -1,23 +1,24 @@
-# Summary
+## Description
 
-- This pr solves **this**
-- And adresses **that**
+### Added
 
-## This
+-
 
-More details on why this was solved and how
+### Changed
 
-## That
+-
 
-How was **that** adressed
+### Fixed
+
+-
 
 ## Testing
 
 ### How to prepare for test
 
 - [ ] ssh to hasta (depending on type of change)
 - [ ] install on stage:
-`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-housekeeper-stage.sh [THIS-BRANCH-NAME]`
+`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_main -t housekeeper -b YOURBRANCH`
```

Comment:
```suggestion
```bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_main -t housekeeper -b YOURBRANCH```
```
If we have 3 - then markdown shows the copy symbol and you do not have to mark the text

Reply:
done 👍